hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -87,8 +92,8 @@ healthyCloseUTxO =
   (genOneUTxOFor somePartyCardanoVerificationKey `suchThat` (/= healthyUTxO))
     `generateWith` 42
 
-healthySnapshotNumber :: SnapshotNumber
-healthySnapshotNumber = 1
+genSnapshotNumber :: SnapshotNumber
+genSnapshotNumber = genSnapShot `generateWith` 42
```

Comment:
Beware: this does not generate arbitrary values on different test runs. It's basically just picking some random number, which might change whenever we change the code, but not multiple values within one version of the code.

Reply:
I've reverted this in the end anyway.